fluentui_peoplepicker/src/main/java/com/microsoft/fluentui/tokenized/peoplepicker/PeoplePicker.kt

```diff
@@ -194,7 +194,6 @@ fun PeoplePicker(
                             )
                             Spacer(modifier = Modifier.width(chipHorizontalSpacing))
                         }
-                        focusRequester.requestFocus()
                     }
```

Comment:
as this call has not been used then remove the focusRequester code from modifier & import as well.

Reply:
My bad, somehow changes dint add to this PR. Resolved